sidekick/pubspec.lock

```diff
@@ -434,7 +434,7 @@ packages:
       name: sidekick_core
       url: "https://pub.dartlang.org"
     source: hosted
-    version: "0.13.0"
+    version: "0.13.1"
```

Comment:
I think we should adjust the `sidekick --version` test.
- Either don't check the `sidekick_core` version or 
- don't override the dependency to use the local path